torch/csrc/distributed/c10d/socket.cpp

```diff
@@ -413,22 +453,34 @@ bool SocketImpl::waitForInput(std::chrono::milliseconds timeout) {
     int res = pollFd(&pfd, 1, static_cast<int>(timeout.count()));
     if (res > 0) {
       return true;
+    } else if (res == 0) {
```

Comment:
Nit:
You can just write but not a big deal as compiler is going to do this anyway :-)
```suggestion
    } 
    if (res == 0) {
```

Reply:
I'm just going to leave it as is -- I prefer the else even if it doesn't do anything semantically in this case. I think it's clearer to understand that these are mutually exclusive